ansible/roles/sufia/tasks/data-repo.yml

```diff
@@ -64,6 +64,6 @@
     ignore_errors: True
 
   become: yes
-  become_user: '{{ project_user }}'
+  become_user: '{{ project_runner }}'
```

Comment:
It looks like geoblacklight is using `project_owner` as appropriate.

I will need to get a sample `user_list.txt`, `admin_list.txt` and `images.zip` from you for future testing.